docs/conf.py

```diff
@@ -238,7 +241,91 @@
                       'issue '),
             'fileref': ('https://github.com/NVIDIA/DALI/tree/' + (git_sha if git_sha != u'0000000' else "master") + '/%s', ''),}
 
+
+from typing import (
+    Any, Callable, Dict, Iterator, List, Optional, Sequence, Set, Tuple, Type, TypeVar, Union
+)
+from typing import get_type_hints
+
+
+_dali_enums = ["DALIDataType", "DALIIterpType", "DALIImageType", "PipelineAPIType"]
+
+class EnumDocumenter(ClassDocumenter):
+    # Register as .. autoenum::
+    objtype = 'enum'
+    # Produce .. py:class:: fields in the RST doc
+    directivetype = 'class'
+    # priority = 10
+
+    def __init__(self, *args: Any) -> None:
+        super().__init__(*args)
+
+    @classmethod
+    def can_document_member(cls, member: Any, membername: str, isattr: bool, parent: Any
```

Comment:
Do you need to use input/output annotations here? I don't see that in other methods/functions.

Reply:
Leftover when I was overriding sphinx code. Will remove them.